main.js

```diff
@@ -0,0 +1,96 @@
+var Board = function(numPegs=3, numDiscs=5) {
+  this.numPegs = numPegs,
+  this.numDiscs = numDiscs,
+  this.layout = [];
+  for (let index = 0; index < numPegs; index++) {
+    this.layout.push([]);    
+  };
+  for (let index = 0; index < numDiscs; index++) {
+    this.layout[0].push(index + 1);    
```

Comment:
This logic is reversing the correct default state of the board (should be 5...1, not 1...5)